package.json

```diff
@@ -55,7 +55,7 @@
     "eslint-plugin-jsx-a11y": "6.9.0",
     "eslint-plugin-react": "7.34.4",
     "eslint-plugin-react-hooks": "4.6.2",
-    "husky": "9.1.1",
+    "husky": "9.0.11",
```

Comment:
I think I only notice now, but is there a reason to downgrade the husky version ?

Reply:
Maybe while rebasing, it accidentally downgraded.